sql/5_0_2-to-6_0_0_upgrade.sql

```diff
@@ -2406,7 +2407,6 @@ ALTER TABLE `oauth_clients` ADD `logout_redirect_uris` text;
 
 #IfMissingColumn oauth_trusted_user grant_type
 ALTER TABLE `oauth_trusted_user` ADD COLUMN `grant_type` varchar(32) DEFAULT NULL;
-#EndIf
```

Comment:
will need to add this line back (could just use the cheap trick to bring in this original file from master to make this disappear on the diff also :) )